modules/apps/data-engine/data-engine-lang/src/main/resources/content/Language.properties

```diff
@@ -63,6 +64,7 @@ there-are-no-results-with-these-attributes=There are no results with these attri
 there-are-no-rules=There are no rules.
 there-are-no-rules-description=Define conditions and actions to dynamically change fields and elements in the form.
 this-action-may-erase-data-permanently=This action may erase data permanently.
+to-delete-you-first-need-to-remove-it-from-the-following-structures=To delete you first need to remove it from the following structures:
```

Comment:
When purpose comes at the beginning, it needs to be separated by comma. Example: To do something, I must do another thing.